### PR TITLE
Audit fix and add repository field to package.json in Lens

### PR DESCRIPTION
**Pull request by tacigar**
* Do `audit fix`
* Add repository field to package.json

### CODE REVIEW

File: zipkin-lens/package.json [hunk 1]
```diff
@@ -9,6 +9,10 @@
     "build": "cross-env NODE_ENV=production webpack -p --config ./webpack.prod.config.js",
     "start": "cross-env NODE_ENV=development webpack-dev-server -d --config ./webpack.dev.config.js"
   },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/openzipkin/zipkin"
+  },
```
Comment (tacigar) [inline zipkin-lens/package.json:15]: Add repository field since warning occurred when `npm install` was executed.

**Comment by codefromthecrypt** [comment]
weird.. fails on lint

**Comment by tacigar** [comment]
I'll fix them... 😢

**Comment by tacigar** [comment]
@adriancole 
Thank you!!

**Comment by codefromthecrypt** [comment]
no prob!

**Pull request merged**